server/flaskr/services/notifications/handlers/meeting_started_notification_handler.py

```
@@ -0,0 +1,57 @@
+}
+
+
+class MeetingStartedNotificationHandler:
+    def create(self, orgId, meetingId: int):
+        return NotificationJob(
+            -1,
+            orgId,
+            meetingId,
+            NotificationType.MeetingStarted.value,
+            "",
+            datetime.now(),
```

Ruff (0.15.21)

[warning] `datetime.datetime.now()` called without a `tz` argument

(DTZ005)